src/app/CMakeLists.txt

```diff
@@ -108,6 +108,11 @@ if (AU_MODULE_EFFECTS_LV2)
     add_definitions(-DAU_MODULE_EFFECTS_LV2)
 endif()
 
+if (MUSE_MODULE_AUTOBOT)
+    list(APPEND LINK_LIB muse::autobot)
+    add_definitions(-DMUSE_MODULE_AUTOBOT)
```

Comment:
it shouldn't be necessary, to use define, we need to include `muse_framework_config.h`, it's there